backend/src/db/seed.js

```diff
@@ -553,7 +553,7 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
     ])
     authenticatedUser = null
 
-    const comments = await Promise.all([
+    const [trollingCommentC1, trollingCommentC2] = await Promise.all([
       factory.create('Comment', {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-01T14:11:48Z" title="Saturday, February 1st 2020, 3:11:48 pm +01:00">Feb 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -610,7 +610,8 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
         postId: 'p15',
       }),
     ])
-    const trollingComment = comments[0]
+    const trollingCommentC1 = comments[0]
+    const trollingCommentC2 = comments[1]
```


```suggestion
    const [ trollingCommentC1, trollingCommentC2] = comments
```

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T12:59:12Z" title="Wednesday, February 5th 2020, 1:59:12 pm +01:00">Feb 5, 2020</time>_

---


Yes, good seen @roschaefer 😍